test/test_ftp_object.jl

```diff
@@ -1,171 +1,325 @@
-using FTPClient
+@testset "FTPObject" begin
+
+    expected_header_port = r"229 Entering Extended Passive Mode \(\|\|\|\d*\|\)"
```

Comment:
I think it's ok to have it declared in runtests.jl and use it but I understand the concern. Once per file is a big improvement already so I am happy.